AUTHORS

```diff
@@ -412,19 +461,29 @@ Wang Li                                 wangli09@kuaishou.com
 Wayne Haber                             whaber@gitlab.com
 Wazir Ahmed                             wazir@accuknox.com
 Weilong Cui                             cuiwl@google.com
+Wei Yang                                wei.yang@daocloud.io
 Weizhou Lan                             weizhou.lan@daocloud.io
+wenhuwang                               wang15691700816@gmail.com
```

Comment:
‪Wenhui Wang?

Reply:
not sure where the `i` comes from, but `Wenhu Wang` seems like a reasonable approximation.